src/lib.rs

```diff
@@ -0,0 +1,408 @@
+                let name = "<unknown>";
+                let version = None;
+                let (current_crate, baseline_crate) = generate_versioned_crates(
+                    &mut config,
+                    CurrentCratePath::CurrentRustdocPath(current_rustdoc_path),
+                    &*loader,
+                    &rustdoc_cmd,
+                    name,
+                    version,
+                )?;
+
+                let success = run_check_release(&mut config, name, current_crate, baseline_crate)?;
+                vec![Ok(success)]
+            }
+            RustdocSource::Root(project_root) => {
+                let metadata = manifest_metadata(project_root)?;
+                let selected = self.scope.selected_packages(&metadata);
+                selected
+                    .iter()
+                    .map(|selected| {
+                        let manifest_path = selected.manifest_path.as_std_path();
+                        let crate_name = &selected.name;
+                        let version = &selected.version;
+
+                        let is_implied = matches!(
+                            self.scope.mode,
+                            ScopeMode::DenyList(PackageSelection {
+                                selection: ScopeSelection::Workspace,
+                                ..
+                            })
+                        ) && selected.publish == Some(vec![]);
+                        if is_implied && selected.publish == Some(vec![]) {
+                            config.verbose(|config| {
+                                config.shell_status(
+                                    "Skipping",
+                                    format_args!("{crate_name} v{version} (current)"),
+                                )
+                            })?;
+                            Ok(true)
+                        } else {
+                            config.shell_status(
+                                "Parsing",
+                                format_args!("{crate_name} v{version} (current)"),
+                            )?;
+
+                            let (current_crate, baseline_crate) = generate_versioned_crates(
+                                &mut config,
+                                CurrentCratePath::ManifestPath(manifest_path),
+                                &*loader,
+                                &rustdoc_cmd,
+                                crate_name,
+                                Some(version),
+                            )?;
+
+                            Ok(run_check_release(
+                                &mut config,
+                                crate_name,
+                                current_crate,
+                                baseline_crate,
+                            )?)
+                        }
+                    })
+                    .collect()
+            }
+            RustdocSource::Revision(_, _) | RustdocSource::Version(_) => todo!(),
```

Comment:
I believe these are currently unimplemented in the CLI, so I'm fine with leaving a panic in this branch.

But let's make it `unimplemented!()` with a suitable message instead of a bare `todo!()`, which makes it seem like we forgot to add it into the PR rather than making an explicit choice to not include it for now.

Reply:
After the changes from [tonowak](https://github.com/tonowak) the `todo` is not there anymore 👍